testdata/oracle/procedure/invoke-stored-procedure.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Alternatively, the procedure can be invoked with the key word `EXEC` instead of embedding it winto `BEGIN` `END` block.

Reply:
new test case i[nvoke-stored-procedure-with-exec](https://github.com/DataDog/go-sqllexer/pull/26/files#diff-da88539a43ed72c4c8d7d61f73c8acba04ce08213966e476c3e9c50c9da2dddd) added